psydac/api/ast/parser.py

```diff
@@ -432,6 +432,7 @@ def _visit_DefNode(self, expr, **kwargs):
 
         if self.backend['name'] == 'pyccel':
             a = build_types_decorator(arguments)
+            a = [String(str(i)) for i in a]
```

Comment:
Can we merge these two lines into one? This way we avoid redefining the variable `a`.